lib/rails_api_auth.rb

```diff
@@ -41,4 +41,9 @@ module RailsApiAuth
   # currently only accepts :uuid
   mattr_accessor :primary_key_type
 
+  # @!attribute [rw] force_ssl
+  # Force SSL for Oauth2Controller.
+  mattr_accessor :force_ssl
```

Comment:
yes, what documentation tool is this? (I just copied from another attribute, but have no idea what tool this actually is)